pyhomematic/_hm.py

```diff
@@ -680,8 +680,12 @@ def proxyInit(self):
             LOG.debug("ServerThread.proxyInit: init('http://%s:%i', '%s')" %
                       (callbackip, callbackport, interface_id))
             try:
+                import debugpy
+                debugpy.breakpoint()
                 # at least for home ip, init ccu is not working. Read list devices before "init proxy"
-                if interface_id == "homeassistant-HMIP":
+                # this fix is only applied for HmIP-RF (port 2010), as other interfaces not always implement the listDevice function
+                # and do not show this issue
+                if proxy._remoteport == 2010:
```

Comment:
I'd prefer `if proxy._remoteport in [2010, 32010, 42010]`. That would cover all possible configurations.

Reply:
understood - changed accordingly.